### PR TITLE
configure PyPI long_description

### PR DESCRIPTION
**Pull request by graingert**
PyPI requires reStructuredText.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,10 +1,14 @@
 from setuptools import setup
 
+readme = open('README.rst').read()
+
 setup(
     name="nbval",
     version="0.5",
     author="Laslett, Cortes, Kluyver, Pepper, Fangohr",
+    author_email="cmg@soton.ac.uk",
```
Comment (graingert) [inline setup.py]: needed for `check -r -s`

**Comment by takluyver** [comment]
Thanks! It's failing on Travis, however, because it tries to run the check but doesn't have docutils installed - can you fix that one way or another?

File: .travis.yml [hunk 2]
```diff
@@ -1,27 +1,20 @@
+sudo: false
 language: python
-
 python:
   - "nightly"
+  - "3.7"
```
Comment (takluyver) [inline .travis.yml]: This doesn't seem to be working.

**Comment by takluyver** [comment]
On Python 3.4 and 2.7, check gives:

```
running check
warning: check: Could not finish the parsing.
error: Please correct your package.
```

Maybe separate the check out of the `make test` target and just run it on Python 3.6?

**Comment by takluyver** [comment]
PyPI recently gained the ability to accept Markdown, so I've taken the lazy option here. Thanks!

**Comment by takluyver** [comment]
I messed up the upload by building the package with an older version of setuptools that doesn't support `long_description_content_type`, so the markdown isn't actually rendered correctly. I'd forgotten that it wouldn't complain about an argument it didn't know.

**Pull request closed without merging**